app/src/main/res/raw/data.json

```diff
@@ -995,6 +1091,17 @@
             "topics": ["животные"]
         }}
     ],
+    [
+        {"EN":{
+            "word": "alcohol",
+            "topics": ["cooking and eating", "food"],
+            "level": "intermediate(B1)"
+        }},
+        {"RU": {
+            "word": "алкоголь",
```

Comment:
In my opinion we need to remove all alcohol because of rating. And also it is not food.

Reply:
Discussed offline - removed